Source/Bifrost.Events.Azure.Tables/EventStore.cs

```diff
@@ -71,7 +71,7 @@ public void Commit(IEnumerable<EventAndEnvelope> eventsAndEnvelopes)
                 batch.Add(TableOperation.Insert(@event));
             });
 
-            _table.ExecuteBatchAsync(batch);
+            _table.ExecuteBatchAsync(batch).Wait();
```

Comment:
If the eventstore table does not exist, the subsequent call to commit can fail because the table does not exist yet?